content/2020/2020-10/2020-10-23-pulsar-weekly.md

```diff
@@ -0,0 +1,143 @@
+---
+title: "2020-10-23-pulsar-weekly"
+displayTitle: "Pulsar Community Weekly Update - 2020-10-17 ~ 2020-10-23"
+date: 2020-10-17 ~ 2020-10-23
+description: "This is the Pulsar community weekly update for 2020-10-17 ~ 2020-10-23, with updates on Pulsar client, broker, Pulsar Functions, transaction, tiered storage, and so on."
+id: "2020-10-23-pulsar-weekly"
+---
+
+## [Pulsar Community Weekly Update] 2020-10-17 ~ 2020-10-23
+
+This is the Pulsar community weekly update for 2020-10-17 ~ 2020-10-23, with updates on Pulsar client, broker, Pulsar Functions, transaction, tiered storage, and so on.
+
+### Pulsar Highlight
+
+Pulsar has supported acknowledging the batch message index since release 2.6.0. But Pulsar does not support setting the cursor. If users reset the cursor to a batch index of the batch message when consuming messages again, they will get the batch index again. Now, Pulsar supports resetting the cursor to a batch index of the batch message.
```

Comment:
Pulsar supports resetting the cursor to a batch index of the batch message now.
In 2.6.0 release, Pulsar supports acknowledging the batch message index, yet you cannot set the cursor. If you reset the cursor to a batch index of the batch message when consuming messages again, you will get the batch index again. Now you can reset the cursor now.

Reply:
1. We can tell readers the highlights first.
2. Try to use present tense if possible.